src/string/lipogram.rs

```diff
@@ -0,0 +1,107 @@
+//*
+// Fn that checks if the slice is a lipogram
+// Lipogram - sentence that exclude one or more letters from the alphabet
+//
+// if you only need one result use is_lipogram(str).0 for bool or use is_lipogram(str).1 for String
+pub fn is_lipogram(lipogram_str: &str) -> (bool, String) {
+    let alphabet = "abcdefghijklmnopqrstuvwxyz";
+    let mut excluded_letters = vec![];
+
+    if lipogram_str.is_empty() {
+        return (false, String::from("The sentence is empty"));
+    }
+
+    for letter in alphabet.chars() {
+        if !lipogram_str.to_lowercase().contains(letter) {
+            excluded_letters.push(letter);
+        }
+    }
```

Comment:
This is inefficient. Could be done in _linear time_ using some kind of _hashing_.